src/linear_solver_stops/stop_ma_thres.jl

```diff
@@ -0,0 +1,126 @@
+# This file is part of RLinearAlgebra.jl
+
+"""
+    LSStopMA <: LinSysStopCriterion
+
+A structure that specifies a stopping criterion that incoroporates the randomness of the moving average estimator. That is, once a method
+    achieves a certain number of iterations, it stops.
+
+# Fields
+- `max_iter::Int64`, the maximum number of iterations.
+- `threshold::Float64`, the value of the estimator that is sufficient progress. 
+- `delta1::Float64`, the percent below the threshold does the true value of the progress estimator need to be for not stopping to be a mistake. This is equivalent to stopping too late.
+- `delta2::Float64`, the percent above the threshold does the true value of the progress estimator need to be for stopping to be a 
+    mistake. This is equivalent to stopping too early.
+- `chi1::Float64`, the probability that the stopping too late action occurs.
+- `chi2::Float64`, the probability that the stopping too early action occurs.
+"""
+mutable struct LSStopMA <: LinSysStopCriterion
+    max_iter::Int64
+    threshold::Float64
+    delta1::Float64
+    delta2::Float64
+    chi1::Float64
+    chi2::Float64
+end
+
+LSStopMA(iter; 
```

Comment:
doctstring does not include constructor?

Reply:
I added the constructor.